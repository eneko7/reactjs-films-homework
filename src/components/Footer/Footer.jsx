import React from 'react';
import style from './Footer.scss';
import Signature from '../Signature';

class Footer extends React.Component {
    render() {
        return(
            <footer className={style.main_container_footer}>
                <span className={style.main_container_footer_films}>FILMS</span>
                <span className={style.main_container_footer_copyright}>Copyright Â© {(new Date()).getFullYear()} FILMS. <Signature name="IHAR KARPUK" /></span>
            </footer>
        );
    }
}

export default Footer;