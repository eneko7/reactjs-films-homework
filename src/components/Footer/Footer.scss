.main_container_footer{
    width:100%;

    .main_container_footer_films{
        font-size: 38px;
        color: #0aaee4;
        font-family: "Helvetica Neue Roman";
        font-weight: 800;
        text-align: center;
        display: block;
        margin: 40px auto 40px auto;
    }
    .main_container_footer_copyright{
        font-size: 20px;
        color: #969797;
        font-family: "Helvetica Neue Roman";
        text-align: center;
        display: block;
        margin: 0px auto 40px auto;
    }
}