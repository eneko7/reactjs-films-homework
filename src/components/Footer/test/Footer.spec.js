import React from 'react';
import renderer from 'react-test-renderer';

import Footer from '../index';

describe('Footer', () => {
    it('Footer renders correctly', () => {
        const renderedFooter = renderer.create(<Footer />).toJSON();
        expect(renderedFooter).toMatchSnapshot();
    });
});