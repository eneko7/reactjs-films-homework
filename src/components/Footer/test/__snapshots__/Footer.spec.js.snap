// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Footer renders correctly 1`] = `
<footer
  className="main_container_footer"
>
  <span
    className="main_container_footer_films"
  >
    FILMS
  </span>
  <span
    className="main_container_footer_copyright"
  >
    Copyright Â© 
    2018
     FILMS. 
    <span
      className="main_container_footer_copyright_signature"
    >
      IHAR KARPUK
    </span>
  </span>
</footer>
`;
